packages/opentelemetry-exporter-jaeger/test/jaeger.test.ts

```diff
@@ -150,5 +151,17 @@ describe('JaegerExporter', () => {
         assert.strictEqual(result, ExportResult.SUCCESS);
       });
     });
+
+    it('should use httpSender if config.endpoint is setten and set x-opentelemetry-outgoing-request header', () => {
+      const exporter = new JaegerExporter({
+        serviceName: 'opentelemetry',
+        endpoint: 'http://testendpoint',
+      });
+      assert.strictEqual(exporter['_sender'].constructor.name, 'HTTPSender');
+      assert.strictEqual(
+        exporter['_sender']._httpOptions.headers[OT_REQUEST_HEADER],
+        1
+      );
```

Comment:
Could add a test that mock the endpoint and verify that the exporter actually send the correct data ?

Reply:
Added nock at: 5620e16609b20bd9e7b7e4be82e5576c41035de8. So I could mock the endpoint.

Added tests that check if the header was sent to the endpoint at: 90f919e28f9fb509c7fe9fc37d4ee3b8ee9d3ceb.